packages/cli/src/controller/deploy-controller.ts

```diff
@@ -161,12 +160,20 @@ export async function getEndpoint(chainId: string, url: string): Promise<string>
         baseURL: url,
       })
     ).data;
-    return result.find((endpoint: endpointType) => endpoint.chainId === chainId).endpoint;
+    return result;
   } catch (e) {
     errorHandle(e, 'Failed to get endpoint:');
   }
 }
 
+export function processEndpoints(endpoints: endpointType[], chainId: string): null | string {
```

Comment:
Be careful with `null` and `undefined`. They are not the same thing

```suggestion
export function processEndpoints(endpoints: endpointType[], chainId: string): undefined | string {
```